js/datagrid.ts

```diff
@@ -321,25 +287,48 @@ export class DataGridView extends DOMWidgetView {
     this.grid.cellClicked.connect(
       (sender: FeatherGrid, event: FeatherGrid.ICellClickedEvent) => {
         if (this.model.comm) {
-          this.model.comm.send(
-            {
-              method: 'custom',
-              content: {
-                event_type: 'cell-click',
-                region: event.region,
-                column: event.column,
-                column_index: event.columnIndex,
-                row: event.row,
-                primary_key_row: event.primaryKeyRow,
-                cell_value: event.cellValue,
-              },
-            },
-            null,
-          );
+          this.send({
+            event_type: 'cell-click',
+            region: event.region,
+            column: event.column,
+            column_index: event.columnIndex,
+            row: event.row,
+            primary_key_row: event.primaryKeyRow,
+            cell_value: event.cellValue,
+          });
         }
       },
     );
 
+    this.model.data_model.dataSync.connect((sender, msg) => {
```

Comment:
Be careful with this one. It seems that every view will now send a cell-changed message, and update the model, when the model changes?

Since this is hooking into a model signal, this is probably better to do at the model level. If you wanted to send the message about the cell change with a view's callbacks, perhaps that message should be sent from the view rather than here?

Reply:
So basically use the view instance from the model when sending the message?

Reply:
I think it's trickier, in that the change is not associated with a view at that point. If you can associate the specific cell change with a view, then you can send the message from the view.